src/features/components/templates/navigation/main-nav/content/platform/index.ts

```diff
@@ -166,6 +167,23 @@ export const tradingPlatformsContent: SmartNavContent[] = [
             is_eu: true,
         },
     },
+    {
+        id: 5,
+        data: {
+            aria_label: 'Deriv cTrader',
+            icon: {
+                src: DerivcTrader,
+                alt: 'Deriv cTrader trading platform',
```

Comment:
Base on SEO team request we need to translate our alt messages too, please use `_t_` here and add localise function in the component for the alt message :pray: